beetsplug/discogs.py

```diff
@@ -218,25 +234,36 @@ def get_album_info(self, result):
         # information and leave us with skeleton `Artist` objects that will
         # each make an API call just to get the same data back.
         tracks = self.get_tracks(result.data['tracklist'])
-        albumtype = ', '.join(
-            result.data['formats'][0].get('descriptions', [])) or None
-        va = result.data['artists'][0]['name'].lower() == 'various'
+
+        # Extract information for the optional AlbumInfo fields, if possible.
+        va = result.data['artists'][0].get('name', '').lower() == 'various'
```

Comment:
This line looks a bit off, as still accesses a key directly and assumes it has at least one item on the list - but it should be safe to assume it will be present if the execution reaches this point (as the sanity check has passed, and the call to `get_artist()` was completed successfully).